its/combination.ts

```diff
@@ -1,12 +1,25 @@
 import { PipelineCombination } from "./types";
 
 export function generateCombinations(): PipelineCombination[] {
-  const osCombinations: PipelineCombination["os"][] = ["unix", "windows"];
-  const versionCombinations: PipelineCombination["version"][] = [
-    { extension: "sonarcloud", version: 1 },
-    { extension: "sonarcloud", version: 2 },
-    { extension: "sonarcloud", version: 3 },
-  ];
+  let osCombinations: PipelineCombination["os"][];
+  switch(process.env.RUN_ON_OS) {
```

Comment:
I don't understand where those env variables are coming from or where they are defined. How did you find it?

Reply:
I used to pass them from the Azure Pipeline when I tried the matrix job. Then I reverted to a single job (see details in the JIRA ticket) but I decided to keep this code in case we want to run only a subset of the tests.